pages/rs_quality/fair.md

```diff
@@ -1,5 +1,169 @@
+- **Interoperable** - when interacting with other software it must be done by exchanging data and/or metadata through
+  standardised protocols and application programming interfaces (APIs).
+- **Reusable** - software should be usable (can be executed) and reusable
+  (can be understood, modified, built upon, or incorporated into other software).
+
+Each of the above principles can be achieved by a number of practices listed below.
+This is not an exact science, and by all means the list below is not exhaustive,
+but any of the practices that you employ in your research software workflow will bring you
+closer to the gold standard of fully reproducible research.
+
+### Findable
+- Create a description of your software to make it discoverable by search engines and other search tools
+- Place your software in a public software repository (and ideally register it in a [general-purpose or domain-specific software registry][software-registries])
+- Use a unique and persistent identifier (DOI) for your software (e.g. by depositing your code on [Zenodo][zenodo]), 
+which is also useful for citations - note that depositing your data/code on GitHub and similar software repositories 
+may not be enough as they may change their open access model or disappear completely in the future, so archiving your code means it stands a better chance at being preserved 
+
+### Accessible
+- Make sure people can freely, legally and easily get a copy your software
+- Structure your code using common patterns and use coding conventions to make your code readable and understandable by people (once they obtain a copy of it), and thereby more accessible
+- The code and its description has to available even when the software is not longer in use. This include previous versions of the software.
+
+### Interoperable
+- Explain the functionality of your software and protocols for interaction with it
+- Use standard formats for inputs and outputs
+- Communicate with other software and tools via standard protocols and APIs
+
+### Reusable
+- Document your software (including its functionality, and how to install and run it) to make it more understandable by others who may wish to reuse or extend it
+- Follow best practices for software development (including structuring your code in reusable functions with a single functionality
+that can be built on top of, coding conventions, code readability and verifying its correctness)
+- Test your software and make sure it works on different platforms/operating systems to make it more reusable
+- Give a licence to your software clearly stating how it can be reused
+- State how to cite your software, so people can give you credit when they reuse it
+- Include a contributor policy so that others can contribute to your code and credit for contributions is provided
+   
+## Tools and practices for FAIR research software development 
```

Comment:
should we add tools that helps the user measure the FAIRness of their software like the FAIRsoft evaluator? https://openebench.bsc.es/observatory/Evaluation

Reply:
Yes, by all means - we need to add more tools and resources to that section.

Reply:
done